zigzag/stages/WorkloadParserStage.py

```diff
@@ -60,6 +61,9 @@ def parse_mapping_data(mapping_yaml_path: str) -> list[dict[str, Any]]:
         mapping_validator = MappingValidator(mapping_data)
         mapping_data = mapping_validator.normalized_data
         mapping_validate_success = mapping_validator.validate()
+        for temp_map in mapping_data[0]["temporal_ordering"]:
```

Comment:
I don't know if it should only do this for mapping_data[0] or all values in mapping_data

Reply:
The items in the `mapping_data` list represent mappings for different layers or operators. So it should be applied to all items.